templates/rustdoc/topbar.html

```diff
@@ -90,21 +90,6 @@
                             </li>
                         </ul>
                     </div>
-
-                    {# Show the crate authors #}
```

Comment:
Could you show the owners instead?

Reply:
I am currently unable to check the topbar related pages because I am having problems with add-essential-files.
See: https://github.com/rust-lang/rustwide/issues/41#issuecomment-804198425

So I don't know if the whole topbar is currently displayed properly. Can you help check it? Or do you have any suggestions for solving the above problem.

Reply:
If you have working IPv6 then https://docs.rs.dev.nemo157.com/ is running this PR currently, looks like it's working to me.

Reply:
Looking at the code, I should have used the correct field. https://github.com/rust-lang/docs.rs/blob/a0438c4bd81459c43640acde8db4f57e8e40db1c/src/web/crate_details.rs#L19

Reply:
> If you have working IPv6 then https://docs.rs.dev.nemo157.com/ is running this PR currently, looks like it's working to me.

Thanks!

Reply:
> I am having problems with add-essential-files.
> See: [rust-lang/rustwide#41 (comment)](https://github.com/rust-lang/rustwide/issues/41#issuecomment-804198425)

@jyn514  @Nemo157  Do you have any suggestions to solving this?